.github/actions/report-job-status/action.yml

```diff
@@ -19,7 +19,8 @@ runs:
         # the team chat. Report failures occurred in PRs and other branches
         # to the personal chat.
         if [[ ${{ github.ref }} == refs/heads/master ||
-              ${{ github.ref }} =~ refs/heads/[0-9].[0-9]+$ ]]; then
+              ${{ github.ref }} =~ refs/heads/[0-9].[0-9]+$ ||
+              ${{ github.actor }} == 'TarantoolBot' ]]; then
```

Comment:
Could you please omit quotes for consistency with others?

Reply:
fixed